### PR TITLE
Add EncryptRawMessageArmored helper function

### PR DESCRIPTION
**Pull request by hcmlinj**
To generates an armored PGP message given a rawdata and an armored public key.

### CODE REVIEW

File: helper/helper.go [hunk 1]
```diff
@@ -69,6 +69,36 @@ func EncryptMessageArmored(key, plaintext string) (ciphertext string, err error)
 	return ciphertext, nil
 }
 
+// EncryptRawMessageArmored generates an armored PGP message given a rawdata and
+// an armored public key.
+func EncryptRawMessageArmored(key string, data []byte) (string, error) {
```
Comment (wussler) [inline helper/helper.go:74]:
```suggestion
func EncryptBinaryMessageArmored(key string, data []byte) (string, error) {
```

File: helper/helper_test.go [hunk 2]
```diff
@@ -49,6 +49,30 @@ func TestArmoredTextMessageEncryption(t *testing.T) {
 	assert.Exactly(t, plaintext, decrypted)
 }
 
+func TestRawMessageArmoredEncryption(t *testing.T) {
```
Comment (wussler) [inline helper/helper_test.go:52]:
```suggestion
func TestArmoredBinaryMessageEncryption(t *testing.T) {
```

File: helper/helper_test.go [hunk 3]
```diff
@@ -49,6 +49,30 @@ func TestArmoredTextMessageEncryption(t *testing.T) {
...
+	plaintext := "Secret message"
+
+	armored, err := EncryptRawMessageArmored(readTestFile("keyring_publicKey", false), []byte(plaintext))
```
Comment (wussler) [inline helper/helper_test.go:55]:
```suggestion
	armored, err := EncryptBinaryMessageArmored(readTestFile("keyring_publicKey", false), plainData)
```

File: helper/helper_test.go [hunk 4]
```diff
@@ -49,6 +49,30 @@ func TestArmoredTextMessageEncryption(t *testing.T) {
 	assert.Exactly(t, plaintext, decrypted)
 }
 
+func TestRawMessageArmoredEncryption(t *testing.T) {
+	plaintext := "Secret message"
```
Comment (wussler) [inline helper/helper_test.go:53]:
```suggestion
	plainData := []byte("Secret message")
```

File: helper/helper.go (hunk 1 above)
Comment (wussler) [inline helper/helper.go:74]: The function contains a lot of duplicated code with `EncryptMessageArmored`, could you create a common internal function that encrypts given the `PlainMessage` and use it in both?

File: helper/helper_test.go [hunk 5]
```diff
@@ -49,6 +49,30 @@ func TestArmoredTextMessageEncryption(t *testing.T) {
...
+
+	armored, err := EncryptRawMessageArmored(readTestFile("keyring_publicKey", false), []byte(plaintext))
+
+	if err != nil {
+		t.Fatal("Expected no error when encrypting, got:", err)
+	}
+
+	assert.Exactly(t, true, crypto.IsPGPMessage(armored))
+
+	decrypted, err := DecryptMessageArmored(
```
Comment (wussler) [inline helper/helper_test.go:63]: Please edit `DecryptMessageArmored`, (like the encrypt function) to have a common internal function with two wrappers:
- `DecryptMessageArmored` with the same signature 
- `DecryptBinaryMessageArmored` returning `[]byte`.

If we expect bytes it makes more sense not to cast the data to a string

**Comment by hcmlinj** [comment]
Hi @wussler ,
I've done the commit according to your suggestion.
Any deficiencies, please let me known.
I'm very happy can make contribution.

**Pull request merged**